src/plugins/home/public/application/components/guided_onboarding/getting_started.tsx

```diff
@@ -36,38 +54,37 @@ const title = i18n.translate('home.guidedOnboarding.gettingStarted.useCaseSelect
   defaultMessage: 'What would you like to do first?',
 });
 const subtitle = i18n.translate('home.guidedOnboarding.gettingStarted.useCaseSelectionSubtitle', {
-  defaultMessage: `Select a guide to help you make the most of your data.`,
+  defaultMessage: `Filter by solution to see related use cases`,
```

Comment:
Should there be a period as there was before?

```suggestion
  defaultMessage: `Filter by solution to see related use cases.`,
```

Reply:
I think this was removed on purpose but happy to put it back -  @kevinsweet what do you think?

Reply:
👍 I'm fine either way - just something I noticed